### ISSUE TITLE
Add a [ Fuzzy AND Waiting ] filter 

### ISSUE THREAD

**Issue by lmwp** [GlotPress/GlotPress-WP#1304]
When a contributor (let's call him Joe) suggests new translations for "fuzzy" strings from another contributor (let's call him Alex), Joe's new suggestions are taken into pending strings, but Alex's old "fuzzy" suggestions are not deleted. The behavior of this feature is correct, because it gives Alex the possibility to propose a new translation for his "fuzzy" strings as well.
Let's say Joe displays all the "fuzzy" strings in the project, and he proposes a new translation for one of Alex's "fuzzy" strings, his new proposal will be put on "waiting" status (so it will not be displayed in the page since only the "fuzzy" strings are displayed). On the other hand, if other "fuzzy" strings have been corrected by other contributors or by Joe previously, nothing tells Joe.
It should be possible to mark the "fuzzy" strings for which there is at least one other proposal "waiting" for validation in order to warn that the string has already been corrected, and to display together in the same page the strings having the status "waiting" and "fuzzy".
Moreover, this will avoid editors to have several waiting proposals for the same fuzzy string.
In summary:
- add a marker to "fuzzy" strings that have one or more pending proposals
- add a ("waiting" AND "fuzzy") option in the filters

**Comment by ocean90** [comment]
> add a marker to "fuzzy" strings that have one or more pending proposals

This will be handled in #621.

> add a ("waiting" AND "fuzzy") option in the filters

This will be added in #1344.
